TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -75,6 +75,10 @@ additionalProperties:
     'options':
       type: dictionary
 
+    'fusion':
```

Comment:
Right, but I thought `options` was the current way to make this schema extensible?
So Neos can declare the fusion subkey in options and all currently possible subkeys of fusion.